app/controllers/support_interface/data_exports_controller.rb

```diff
@@ -48,7 +48,8 @@ def create
     def download
       data_export = DataExport.find(params[:id])
       data_export.update!(audit_comment: 'File downloaded')
-      send_data data_export.data, filename: data_export.filename, disposition: :attachment
+
+      redirect_to rails_blob_path(data_export.file, disposition: 'attachment')
```

Comment:
Should we check if `data_export.file.attached?` here?

Reply:
I took a slightly separate approach here. If the file doesn't exist, the `.find` call will fail.